src/css/dialog.css

```diff
@@ -0,0 +1,41 @@
+.modal-body td {
```

Comment:
This file shouldn't be committed. You need to double-check your git usage to avoid similar things happening again.